### PR TITLE
[OCPCLOUD-1070] Set --cloud-provider=external for supported platforms

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Includes revendoring after https://github.com/openshift/library-go/pull/895 and https://github.com/openshift/library-go/pull/1077 changes are merged. Requires for KAPI to apply `--cloud-provider=external` flag on supported platforms.

### CODE REVIEW

**Comment by Danil-Grigorev** [comment]
/reopen

This is still a very much required change.

File: go.mod [hunk 1]
```diff
@@ -30,3 +30,7 @@ require (
 	k8s.io/klog/v2 v2.3.0
 	sigs.k8s.io/kube-storage-version-migrator v0.0.3
 )
+
+replace github.com/openshift/library-go => github.com/Danil-Grigorev/library-go v0.0.0-20200917075037-909099bedd91
+
+replace github.com/openshift/api => github.com/Danil-Grigorev/api v0.0.0-20200914130008-956aacd684e2
```
Comment (elmiko) [inline go.mod:36]: shouldn't these be pointing to the openshift/library-go repo after some commit has merged?

**Review by elmiko — COMMENTED** [review]
question about the go.mod

File: go.mod (hunk 1 above)
Comment (Danil-Grigorev) [reply go.mod]: Yep, was previously using my branch for testing.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**